src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react'
+import '../.css/ArticlesList.css'
+import Articles from './Articles'
+import axios from 'axios'
+
+const ArticlesList = () => {
+    const [articles, setArticles] = useState([])
+    const [topics, setTopics] = useState([])
+    const [isLoading, setIsLoading] = useState(false)
+
+    useEffect(() => {
+        setIsLoading(true)
+        axios
+        .get('https://rahul-vital-be-nc-news.onrender.com/api/articles')
+        .then((articlesResponse) => {
+            setArticles(articlesResponse.data.articles)
+        })
+
+        axios
+        .get('https://rahul-vital-be-nc-news.onrender.com/api/topics')
```

Comment:
There's no requirement to get the topics for this ticket, so this can be removed.